### PR TITLE
enha: replace block number with log index

### PR DESCRIPTION
**Pull request by gabriel-aranha-cw**
This PR implements the log index as specified by RAFT, replacing the placeholder block number.

### CODE REVIEW

**Comment by gabriel-aranha-cw** [comment]
closes https://github.com/cloudwalk/stratus/issues/1221

**Review by renancloudwalk — CHANGES_REQUESTED** [review]
please move storage back inside Consensus, we will use it to commit the data

File: chaos/experiments/leader-election.sh [hunk 1]
```diff
@@ -132,7 +132,7 @@ check_leader() {
     # Check the response for specific strings to determine the node status
     if [[ "$response" == *"append_transaction_executions called on leader node"* ]]; then
         return 0 # Success exit code for leader
-    elif [[ "$response" == *"APPEND_SUCCESS"* ]]; then
```
Comment (renancloudwalk) [inline chaos/experiments/leader-election.sh:135]: I think this should remain OR use a specific message, we actually could change this function to probe several orther behaviors

**Review by renancloudwalk — APPROVED** [review]
nice job!

File: chaos/experiments/leader-election.sh (hunk 1 above)
Comment (gabriel-aranha-cw) [reply chaos/experiments/leader-election.sh:135]: I will have to improve these checks when I work on protobuf definitions soon. I will leave it to the next PR

**Pull request merged**